api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -191,18 +192,28 @@ def test_cannot_create_github_repository_due_to_unique_constraint(
 def test_github_delete_repository(
     client: APIClient,
     organisation: Organisation,
+    feature_external_resource: FeatureExternalResource,
     github_configuration: GithubConfiguration,
     github_repository: GithubRepository,
+    mocker,
```

Comment:
`from pytest_mock import MockerFixture`

```suggestion
    mocker: MockerFixture,
```

Reply:
Added